packages/frontend-shared/src/components/ResultCounts.vue

```diff
@@ -21,14 +25,15 @@
 <script lang="ts" setup>
 
 export interface ResultCountsProps {
-  totalPassed: number|string|null
-  totalFailed: number|string|null
-  totalPending: number|string|null
-  totalSkipped: number|string|null
+  totalPassed?: number|string|null|SpecDataAggregate
```

Comment:
```suggestion
  totalPassed?: number | string | null | SpecDataAggregate
```

Why are these optional but then below are marked with the non-null `!` assertion? If you are rendering this component, these should be defined no?